dojo/unittests/tools/test_intsights_parser.py

```diff
@@ -0,0 +1,98 @@
+from django.test import TestCase
+
+from dojo.models import Engagement, Product, Test
+from dojo.tools.intsights.parser import IntSightsParser
+
+
+class TestIntSightsParser(TestCase):
+    def get_test(self):
+        test = Test()
+        test.engagement = Engagement()
+        test.engagement.product = Product()
+        return test
+
+    def test_intsights_parser_with_no_vuln_has_no_findings_json(self):
+        with self.assertRaises(ValueError) as context:
+            testfile = open("dojo/unittests/scans/intsights/intsights_zero_vul.json")
+            parser = IntSightsParser()
+            findings = parser.get_findings(testfile, self.get_test())
+            testfile.close()
+            self.assertTrue(
+                "IntSights report contains errors:" in str(context.exception)
+            )
+            self.assertTrue("ECONNREFUSED" in str(context.exception))
```

Comment:
I see in this test that you want to raise an exception if the report is empty.
Is it something that you want?
Other parsers just return an empty `list`. Why do you want to raise a value error?